packages/row/src/react/index.js

```diff
@@ -374,22 +375,27 @@ renderProgress.propTypes = {
 
 // NOTE: the `title` prop clashes with a native html attr so we're exclude it
 //       from being mistakenly used in any child component
-const Row = ({ title, titleTruncated, ...props }) => (
-  <div {...styles.row(props)} {...filterReactProps(props)}>
-    {renderOverlays(props)}
+const Row = ({ title, titleTruncated, ...props }) => {
+  const desktop = useMatchMedia('(min-width: 769px)')
```

Comment:
I at first mis-read this as just `matchMedia`.  `use-` indicates it's a hook.  I saw that later.  Even though it kinda bugs me in the native matchMedia api, but I wonder if changing the hook impl to `return { matches }` to match the native api might be more familiar?

Reply:
perhaps that ^ or change the varName to something that doesn't seem like something you'd name the result of a native matchMedia call, like isDesktop?

Reply:
good call. i'll update